kroxylicious-filters/kroxylicious-record-validation/src/main/java/io/kroxylicious/proxy/filter/validation/validators/bytebuf/BytebufValidators.java

```diff
@@ -58,9 +62,18 @@ public static BytebufValidator jsonSchemaValidator(Map<String, Object> schemaRes
         return new JsonSchemaBytebufValidator(schemaResolverConfig, globalId);
     }
 
+    /**
+     * get validator that validates if a record contains a valid JWS (JSON Web Signature) Signature
+     * @return validator
+     */
+    public static BytebufValidator jwsSignatureValidator(JsonWebKeySet trustedJsonWebKeySet, AllowDeny<String> allowedAndDeniedAlgorithms, String jwsRecordHeaderKey,
+                                                         boolean isContentDetached, boolean failOnMissingJwsRecordHeader) {
```

Comment:
```suggestion
    public static BytebufValidator jwsSignatureValidator(JsonWebKeySet trustedJsonWebKeySet, AllowDeny<String> alogrithms, String jwsRecordHeaderKey,
                                                         boolean isContentDetached, boolean requireJwsRecordHeader) {
```
We get the fact its a combined allow and deny list from the type signature. 
`require` is just as explicit and shorter ;)

Reply:
Done in [`a2e691e` (#2861)](https://github.com/kroxylicious/kroxylicious/pull/2861/commits/a2e691e821547d1b0e5d1bb0c4d04752fe9fcd10) and [`b4c9aa7` (#2861)](https://github.com/kroxylicious/kroxylicious/pull/2861/commits/b4c9aa75c77e5a4c2aa27a7235211968143c96c1)